README.md

```diff
@@ -96,6 +96,36 @@
 
 1. if appropriate, send a Feature Change Announcement email to the clients
 
+# Dev Deployment
+
+## 1. Update the Environment Variables (if necessary)
+
+1. on the dev Buttons server (access using ssh):
+
+   1. update the values in `~/BraveButtons/server/.env`
+
+## 2. Deploy the server on Dev
+
+1. on the dev Buttons server (access using ssh):
+
+   1. cd into the `BraveButtons/server` directory
+
+   1. shut down the server process and ensure that environment variables are not cached: `pm2 kill`
+
+   1. pull the latest code from your branch: `git checkout <your branch> && git pull origin <your branch>`
+
+   1. run the server setup script: `sudo ./setup_server.sh ./.env`
+
+## 3. Verify
+
+1. open the production dashboard and confirm that everything appears to be working normally
+
+1. Verify that the logs look reasonable
+
+   1. on the production Buttons server (access using ssh):
+
+      1. run `pm2 logs`
+
```

Comment:
"production" being used here when it should be "dev". Although, i think the whole verify step can be removed more dev deployment

Reply:
fixed in newest push

Reply:
i removed the entire verify section since it's not necessary